src/contexts/BrowserWorker.js

```diff
@@ -275,6 +278,17 @@ function workerBoilerScript() {
 				return def;
 			}
 			function reject(r, transfers) {
+				if (r instanceof Error) {
+					postMessage({
+						cmd: 'deferred_reject_error',
+						token: data.token,
+						error: {
```

Comment:
Ah yes, though shall probably have to resort to [this](http://stackoverflow.com/a/18391400/21677) as `JSON.stringify(new Error('foo'))` seems to give us nothing.